src/applications/gqrx/remote_control.cpp

```diff
@@ -50,6 +50,9 @@ RemoteControl::RemoteControl(QObject *parent) :
     rc_socket = 0;
 
     // Disable proxy setting detected by Qt
+    // Workaround for https://bugreports.qt.io/browse/QTBUG-58374
+    // Fix: https://codereview.qt-project.org/#/c/186124/
+    // Can be removed once Qt 5.9 is available
```

Comment:
We should probably wrap this in a Qt version check since there will be many years where 5.9 will be available alongside with many legacy versions distributed with linux.